doc/profile_changes_SLE15.md

```diff
@@ -0,0 +1,473 @@
+      <name>sle-module-basesystem</name>
+      <version>15</version>
+      <arch>x86_64</arch>
+    </addon>
+  </addons>
+</suse_register>
+```
+
+For extensions which require a registration code write it into the `<reg_code>`
+tag for the respective extension. See more details in the [Module and Extension
+Dependencies](#module-and-extension-dependencies) section.
+
+### Using the Packages DVD Medium
+
+For using a physical Packages DVD medium use this XML snippet:
+
+```xml
+<add-on>
+  <add_on_products config:type="list">
+    <listentry>
+      <media_url><![CDATA[dvd:///]]></media_url>
+      <product>sle-module-basesystem</product>
+      <product_dir>/Module-Basesystem</product_dir>
+    </listentry>
+  </add_on_products>
+</add-on>
+```
+
+*Note: The `product` name must match the internal product name contained in the
+repository. If the product name does not match at installation AutoYaST
+will report an error.*
+
+If you have multiple physical DVD drives you can select a specific device
+using `devices` parameter in the URL, e.g. `dvd:///?devices=/dev/sr1`.
+
+### Using the Packages Medium from a Local Server
+
+You can share the DVD content on the local network via a NFS, FTP or HTTP server.
+
+In that case use the same XML snippet as above, just edit the `media_url`
+tag so it points to root of the medium on the server.
+
+
+### Renamed Software Patterns
+
+The software patterns have been also changed in SLE15. Some patterns have been
+renamed, a short summary is in the following table.
+
+| Old SLE12 Pattern | New SLE15 Pattern |
+| :---------------- | :---------------- |
+| Basis-Devel       | devel_basis       |
+| gnome-basic       | gnome_basic       |
+| Minimal           | enhanced_base     |
+| printing          | print_server      |
+| SDK-C-C++         | devel_basis       |
+| SDK-Doc           | technical_writing |
+| SDK-YaST          | devel_yast        |
+
+#### Notes
+
+- The pattern renames in the table above are not 1:1 replacements, the content
+  of some patterns has been changed as well, some packages were moved to
+  a different pattern or even removed from SLE15.
+- Check that the required packages are still included in the used patterns,
+  optionally use the `<packages>` tag to specify the required packages.
+- The list might be incomplete, some products have not been released for SLE15 yet.
+
+## Registration
+
+### Module and Extension Dependencies
+
+<!--
+Copied from the latest development online documentation:
+https://susedoc.github.io/doc-sle/develop/SLES-autoyast/single-html/#CreateProfile.Register.Extension
+-->
+
+Since SLES 15 AutoYaST automatically reorders the extensions according to their
+dependencies during registration. That means the order of the extensions in
+the AutoYaST profile is not important.
+
+Also AutoYaST automatically registers the dependent extensions even though they
+are missing in the profile. That means you are not required to fill the
+extensions list completely.
```

Comment:
We also can suggest to use SUSEconnect tool to check module dependencies here

Reply:
:heavy_check_mark: Added below